includes/subscriptions/class-wc-payments-subscriptions-event-handler.php

```diff
@@ -121,6 +122,9 @@ public function handle_invoice_paid( array $body ) {
 			$order->payment_complete();
 		}
 
+		// Add the payment intent data to the order.
+		$this->subscription_service->get_and_attach_intent_info_to_order( $order, $wcpay_intent_id );
```

Comment:
I'm wondering if this function should be moved to the `invoice_service` class instead of the `subscription_service` 🤔 

Stripe Billing invoices are represented as WC orders in WooCommerce so I'm thinking it would make sense to call this as `invoice_service->get_and_attach_intent_info_to_order()`. Thoughts?

Reply:
Makes complete sense to me 🙂  updating it